Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -320,6 +320,23 @@ static NSString *const STPSDKVersion = @"13.0.3";
 
 @end
 
+#pragma mark Payment Intents
+
+/**
+ Stripe extensions for working with PaymentIntent objects.
```

Comment:
Looks like the docs for the other extension interface definitions in this file say "STPAPIClient extensions..."